dev-python/pydocstyle/pydocstyle-5.0.2-r1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="Python docstring style checker"
+HOMEPAGE="https://github.com/PyCQA/pydocstyle/"
+SRC_URI="https://github.com/PyCQA/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/snowballstemmer[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}"
```

Comment:
Should remove the `DEPEND` and move the `${RDEPEND}` to `BDEPEND`

Reply:
done